packages/desktop-client/src/components/Modals.tsx

```diff
@@ -28,6 +29,15 @@ import ScheduleDetails from './schedules/EditSchedule';
 import ScheduleLink from './schedules/LinkSchedule';
 import PostsOfflineNotification from './schedules/PostsOfflineNotification';
 
+export type CommonModalProps = {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏thanks! This will be useful when porting the other modals to TS. But would you mind moving this type to `modals.types.ts` or other such file? This will help us get rid of a circular dependency once the other modals are migrated (i.e. modal X imports `CommonModalProps` from `Modals.tsx`, but `Modals.tsx` also imports modal X)

Reply:
Good call, thanks for catching that.